### PR TITLE
[SAGE-457] Sage docs cleanup

### PR DESCRIPTION
**Pull request by monkeypox8**
## Description
Covers a laundry list of documentation site validation/accessibility/performance items to hold us over until the docs redesign.

### Home
- Fixes broken "skip to content" link
- Removes legacy screenshot [SAGE-721](https://kajabi.atlassian.net/browse/SAGE-721). Verified change with design
- Refactors and replaces hero background image
- Adds hero background image blur effect
- Removes devicons dependency, replacing with inline SVGs

### Search
- Adds missing label on form input

### General
- Fixes meta tag links rendering `%` character
- Corrects missing `alt` attribute in Sage logo (assistant/nav)
- "Invalid" iframe widths of `100%` replaced 
- Displays "skip to content" link when focused
- Removes "prettify" code styling on the majority of pages
- Removes `type="text/javascript"` attribute on scripts

## Screenshots

|  Before  |  After  |
|--------|--------|
|||

## Testing in `sage-lib`
1. View the [docs site homepage](http://localhost:4000/pages/index)
2. Confirm that the screenshot has been removed
3. Verify that the "How It Works" icons appear as expected
4. Scan the page using [WAVE](https://wave.webaim.org/extension/) or [axe](https://www.deque.com/axe/), and verify that any remaining issues are color-contrast related.

## Testing in `kajabi-products`

1. (**N/A**) Documentation site changes only

### CODE REVIEW

File: docs/app/views/pages/index.html.erb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <%= render "application/home_hero" %>
-
-<%= render "application/home_sections" %>
-<%= render "application/home_developer" %>
-<%= render "application/home_code" %>
+<main id="main-content">
```
Comment (pixelflips) [inline docs/app/views/pages/index.html.erb:2]: 🔥

File: docs/app/views/layouts/mocks.html.erb [hunk 2]
```diff
@@ -3,7 +3,6 @@
   <head>
     <%= render "application/meta" %>
     <%= render "application/styles" %>
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/devicons/devicon@master/devicon.min.css">
```
Comment (pixelflips) [inline docs/app/views/layouts/mocks.html.erb:6]: Saves 50k! 👍🏼

**Review by pixelflips — APPROVED** [review]
Great work! This is a fantastic clean-up. Left a couple of minor thoughts/ideas, but no show-stoppers. LGTM! 🔥🔥

File: docs/app/views/application/_home_hero.html.erb [hunk 3]
```diff
@@ -1,13 +1,14 @@
-<div class="docs-hero" style="background-image: url(<%= image_url "hero.jpg" %>);">
-  <%= sage_component SageContainer, { size: "xl", css_classes: "sage-type" } do %>
+<div class="docs-hero">
+  <%= image_tag "hero-background.svg", class: "docs-hero__background", alt: "", loading: "lazy" %>
```
[the remark this answers is not in the document]
Comment (monkeypox8) [reply docs/app/views/application/_home_hero.html.erb:2]: Excellent call, I'll make the change

**Review by ju-Skinner — APPROVED** [review]

File: docs/lib/sage-frontend/stylesheets/docs/index.scss [hunk 4]
```diff
@@ -20,7 +20,7 @@
 @import "icon";
 @import "nav";
 @import "panel";
-@import "quick_links";
+// @import "quick_links";
```
Comment (monkeypox8) [inline docs/lib/sage-frontend/stylesheets/docs/index.scss:23]: This was for a legacy feature and can probably be removed from the codebase

Comment (goodwinchris) [reply docs/lib/sage-frontend/stylesheets/docs/index.scss:23]: Lets delete now if un-needed, yeah?

**Review by goodwinchris — APPROVED** [review]
One comment to agree we should removed commented out line in `index.scss` but looks good!

**Pull request merged**